docs/source/model_design/risk_factors.rst

```diff
@@ -1,55 +1,299 @@
+
+Examples:
+
+    * the percentage of people with schistosomiasis parasites in the blood in a village in Kenya in a survey on 12th  December 2019
+    * the proportion of people who have diabetes in China today 
+
++------------------------+-----------------------------------------------+
+|   Measure              | Math                                          |
++------------------------+-----------------------------------------------+
+| Point prevalence       |:math:`\frac{\text{number of current cases in  |
+|                        |a defined population at a point in time}}      |
+|                        |{\text{number of people in the defined         |
+|                        |population at the same time point}}`           |
++------------------------+-----------------------------------------------+
+
+*Period prevalence* is the number of current cases (new and pre-existing) over a defined period of time. The denominator is the average or mid-period population. This measure
+is used when the condition is recurrent and non-fatal. An example of period prevalence would be the proportion of women who have used oral contraceptives at any time
+during the 12-month period preceding the day of the survey.
+
++------------------------+-----------------------------------------------+
+|   Measure              | Math                                          |
++------------------------+-----------------------------------------------+
+| Period prevalence      |:math:`\frac{\text{number of current cases in  |
+|                        |a defined population at over a period of time}}|
+|                        |{\text{average or mid-period population}}`     |
++------------------------+-----------------------------------------------+
+
+Because of these dynamic changes, the magnitude of the prevalence varies from one point in time to another as illustrated by the following diagram:
+
+    .. image:: prevalence_diagram.svg
+
+.. note::
+    In our vivarium models, we estimate the **period prevalence** of condition as
+
+            :math:`\frac{\text{person-time in with-condition state}}{\text{total person time for age, sex, location, year}}`
+   
+
+The *prevalence pool* is the subset of the population who is in the given state (such as the 'with-condition state'). A person who dies from the state is removed from the prevalence pool: death decreases prevalence. People can also exit the prevalence pool by recovering from the state (remission) or emigrating from the population. Diseases with high incidence rates may have low prevalence if they are rapidly fatal or quickly cured. Conversely, diseases with low incidence rates may have substantial prevalence if they are nonfatal but incurable. 

```

Comment:
it's nice to include the clear definition here. I previously thought either a person dies or recovered will be removed from the prevalence state.